index.bs

```diff
@@ -1694,7 +1694,7 @@ on which they appear.  It is language binding specific whether
     <pre highlight="webidl">
         [Exposed=Window]
         interface A {
-          const short rambaldi = 47;
+          const uint16 rambaldi = 47;
```

Comment:
```suggestion
          const int16 rambaldi = 47;
```

This one was signed before. Almost certainly doesn’t matter here for the example, but keeping them the same may help avoid tripping reviewers.